boards/arm/sama5/giant-board/include/board.h

```diff
@@ -0,0 +1,316 @@
+/****************************************************************************
+ * boards/arm/sama5/giant-board/include/board.h
+ *
+ *   Copyright (C) 2015 Gregory Nutt. All rights reserved.
```

Comment:
Since this is a newly created file, shouldn't you put the Apache copyright header?

Reply:
@ghn-certi It's ported from SAMA5D2-XULT, but probably should have an ASF header. Will fix.